OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -599,13 +608,17 @@ void testBushingForce()
 
     osimModel.setGravity(gravity_vec);
 
-    BushingForce spring("bushing", "ground", "ball",
+    auto* spring = new BushingForce("bushing", "ground", "ball",
         transStiffness, rotStiffness, transDamping, rotDamping);
 
-    osimModel.addForce(&spring);
+    osimModel.addForce(spring);
     const BushingForce& bushingForce =
         osimModel.getComponent<BushingForce>("bushing");
 
+    // It's necessary to correct the connectee names in the BushingForce, which
+    // we can do with finalizeConnections() (they are incorrect otherwise
+    // because `spring` is initially orphaned).
+    osimModel.finalizeConnections(osimModel);
```

Comment:
I'm fine with your interpretation. This change was necessary to get the equality comparison to pass. I can investigate if alternate changes to the code can make the test pass. If not, then I will at least update the comment.